### PR TITLE
[FIX] product_cost_usd: inheritance errors related to the old view id t#53862

### PR DESCRIPTION
**Pull request by agarzaarvizu**
## This PR refers to the task[#53862](https://www.vauxoo.com/web#action=1928&cids=1&id=53862&menu_id=1490&model=project.task&view_type=form)

- add pre-migration in order to remove fix errors on inherited views of_product_template_tann_inhrt view

### CODE REVIEW

**Comment by agarzaarvizu** [comment]
@luisg123v Please review.

File: product_cost_usd/migration/11.0.1.0.0/pre-migration.py
Comment (luisg123v) [inline product_cost_usd/migration/11.0.1.0.0/pre-migration.py]: May be removed

File: product_cost_usd/migration/11.0.1.0.0/pre-migration.py
Comment (luisg123v) [inline product_cost_usd/migration/11.0.1.0.0/pre-migration.py]: Missing increasing version

**Review by luisg123v — CHANGES_REQUESTED** [review]
Fix title:
- Not an ADD but a FIX, we're fixing an issue
- Please be more descriptive in titles, I can't know what's being fixed here. Title should say what's being fixed, in the description you can mention the how, which is a migration script

**Comment by luisg123v** [comment]
Hi @andreaarce,

Could you point your change to use this branch, please?

So we know if it's working.

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Review by luisg123v — APPROVED** [review]
LGTM, just waiting for @andreaarce's approval.

FYI you had a typo in title, but since it was only in the PR, I fixed it.

**Review by luisg123v — CHANGES_REQUESTED** [review]
Version increase was lost during rebase.

**Comment by agarzaarvizu** [comment]
@luisg123v Done

**Review by luisg123v — APPROVED** [review]

**Pull request merged**
